subprojects/maven/src/main/java/org/gradle/api/publication/maven/internal/pom/DefaultMavenPom.java

```diff
@@ -133,6 +134,16 @@ public DefaultMavenPom project(Closure cl) {
         return this;
     }
 
+    @Override
+    public DefaultMavenPom project(final Action<? super GroovyObject> action) {
+        return project(new Closure(this, this) {
+            @SuppressWarnings("unused")
+            public void doCall() {
+                action.execute((GroovyObject) getDelegate());
+            }
+        });
+    }
```

Comment:
This should be the other way around: the `Closure` variant calling the `Action` variant. It would result in nicer code, too.